code/modules/events/apc_short.dm

```diff
@@ -26,55 +26,95 @@
 /datum/event/apc_short/end()
 	return TRUE
 
-/proc/power_failure(announce=TRUE)
+/proc/depower_apcs()
 	var/list/skipped_areas_apc = list(
 		/area/engine/engineering,
 		/area/turret_protected/ai)
+	GLOB.event_announcement.Announce("Power failure detected in [station_name()]'s powernet. All APCs have lost power. Gravity systems likely to fail.", "Systems Power Failure", new_sound = 'sound/AI/attention.ogg')
+	for(var/thing in GLOB.apcs)
+		var/obj/machinery/power/apc/A = thing
+		var/area/current_area = get_area(A)
+		if((current_area.type in skipped_areas_apc) || !is_station_level(A.z))
+			continue
+		var/obj/item/stock_parts/cell/C = A.get_cell()
+		if(C)
+			C.charge = 0
+		current_area.power_change()
+	log_and_message_admins("Power has been drained from all APCs.")
 
+/proc/power_failure(announce=TRUE)
```

Comment:
```suggestion
/proc/power_failure(announce = TRUE)
```
Spacing here

Reply:
For some reason, I cannot add my own comments. In checking this file over yet again, I realized there is variable only used once. 

`	var/const/delayed                     = FALSE`
```
/datum/event/apc_short/start()
	power_failure(announce=delayed)
```
I'm going to get rid of it and fix the spacing there as well.